src/utils/utils.js

```diff
@@ -11,3 +11,11 @@ export function getArticles() {
 export function getArticleById(article_id) {
   return newsApi.get(`/articles/${article_id}`);
 }
+
+export function getCommentsByArticle(article_id) {
+  return newsApi.get(`/articles/${article_id}/comments`);
+}
+
+export function postCommentsByArticleId(article_id, newComment) {
+  return newsApi.post(`/articles/${article_id}/comments`, newComment);
```

Comment:
This is me being fussy but I think it's better to extract the info you want here in the api file rather than in the component. Not a mandatory change though!